### ISSUE TITLE
Opt in option for the master tenancy multi-tenancy model to automatically create the master table

### ISSUE THREAD

**Issue by jeremydmiller** [JasperFx/marten#3012]
Issue: User has OptimizeArtifactWorkflow, which will turn off Marten's ability to automatically create the one single tenant database. Make this an explicit opt in part of configuring like so:

```csharp
opts.MultiTenantedDatabasesWithMasterDatabaseTable(ConnectionSource.ConnectionString, "tenants", AutoCreate: AutoCreate.CreateOr*****);
```

**Comment by jeremydmiller** [comment]
What about seeding it with initial tenant databases and connection strings?

**Comment by jeremydmiller** [comment]
Closed by #3020
